### DISCUSSION TITLE
目前可以通过Scene知道新加入的好友的来源，但是当来源是名片来源时，怎么知道是谁推荐了这个名片？

### DISCUSSION THREAD

**Discussion by fangyaoyu** [wechaty/wechaty#2056] [Q&A]
🚨 The issue tracker is not for questions 🚨

If you have a question, please ask it on https://stackoverflow.com/questions/tagged/wechaty

[question]

**Comment by huan** [comment]
I have a question: does WeChat can show the contact name who recommend the card to you?
